readthedocs/search/views.py

```diff
@@ -132,8 +110,7 @@ def get(self, request, project_slug):
             use_advanced_query=use_advanced_query,
         )
 
-        context = self.get_serializer_context(project_obj, user_input.version)
```

Comment:
Passing the context here was saving just one query, and to pass it again we will need to query the version (we have only the slug), so we are just omitting the `projects` parameter here for now (we would use it again when using the new search)